boot.php

```diff
@@ -69,7 +72,7 @@
                 'buster' => time(),
             ];
 
-            return '<div id="rex-quicknavigation-structure" data-url="' . rex_url::currentBackendPage($params + QuickNavigationApi::getUrlParams()) . '"></div>' . $ep->getSubject();
+            return '<div id="rex-quicknavigation-structure" data-url="' . rex_url::currentBackendPage($params + MenuRender::getUrlParams()) . '"></div>' . $ep->getSubject();
```

Comment:
Hier wird man wieder in einen Whoops laufen.
Würde mal anschauen, ob man während des Updates via Installer das unterbinden kann.
Alternativ für die Installer page generell unterbinden.

Reply:
Ok. Gute Idee